files/en-us/web/css/overscroll-behavior/index.html

```diff
@@ -61,7 +61,7 @@ <h3 id="Preventing_an_underlying_element_from_scrolling">Preventing an underlyin
 
 <p>In our <a href="https://mdn.github.io/css-examples/overscroll-behavior/">overscroll-behavior example</a> (see the <a href="https://github.com/mdn/css-examples/tree/master/overscroll-behavior">source code</a> also), we present a full-page list of fake contacts, and a dialog box containing a chat window. </p>
 
-<p><img alt="" src="https://mdn.mozillademos.org/files/15778/example.png" style="border-style: solid; border-width: 1px; display: block; height: 622px; margin: 0px auto; width: 350px;"></p>
+<p><img alt="" src="example.png" style="border-style: solid; border-width: 1px; display: block; height: 622px; margin: 0px auto; width: 350px;"></p>
```

Comment:
```suggestion
<p><img alt="A popup chat window titled 'Active chat', showing a conversation between Chris and Bob. Behind the chat window is a contact list titled 'overscroll-behavior demo'." src="example.png" style="border-style: solid; border-width: 1px; display: block; height: 622px; margin: 0px auto; width: 350px;"></p>
```

Added an alt description.

Reply:
thanks I'll try and remember to keep an eye out for missing or empty alts.